customfieldnames.civix.php

```diff
@@ -346,3 +346,166 @@ function _customfieldnames_civix_civicrm_alterSettingsFolders(&$metaDataFolders
     $metaDataFolders[] = $settingsDir;
   }
 }
+
+/**
+ * Implements hook_civicrm_buildForm().
+ *
+ * @param string $formName
+ * @param CRM_Core_Form $form
+ */
+function _customfieldnames_civix_civicrm_buildForm($formName, &$form) {
+  $templatePath = realpath(dirname(__FILE__)."/templates");
+  if( ($formName == 'CRM_Custom_Form_Group') && ($form->getAction() == CRM_Core_Action::ADD) ) {
+    $templatePath = realpath(dirname(__FILE__)."/templates");
+    $form->add('text', 'table_name', ts('Set Table Name'), '', FALSE);
+    $form->assign('table_name', TRUE);
```

Comment:
you shouldn't need to assign it to the template. that will happen automatically. try removing this line.

Reply:
Done